components/bl-audio-player-component/src/player-components/controls.js

```diff
@@ -0,0 +1,40 @@
+import { usePlayerHandlers } from '../helpers/use-player-handlers';
+import { ControlButton } from './control-button';
+
+export function Controls(props) {
+  const {
+    audioRef,
+    isPlaying,
+    setIsPlaying,
+    index,
+    setIndex,
+    audioUrls,
+  } = props;
+
+  const { skipBack, skipNext } = usePlayerHandlers(index, setIndex, audioUrls, setIsPlaying);
```

Comment:
since there could be two simple functions https://github.com/Backendless/ui-builder-library/pull/12/files#r918914514

I would move them here into the component